locales/de.po

```diff
@@ -18,14 +18,42 @@ msgstr ""
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
 #. Very simple localization
-#: src/puppetlabs/i18n/main.clj:18
+#: src//puppetlabs/i18n/main.clj:18
 msgid "Welcome! This is localized"
 msgstr "Willkommen ! Draußen nur Kännchen"
 
-#: src/puppetlabs/i18n/main.clj:22
+#. Very simple plural system localization
+#: src//puppetlabs/i18n/main.clj:21
+msgid "{0} of beer on the wall."
+msgstr "{0} von Bier an der Wand."
+
+#: src//puppetlabs/i18n/main.clj:22
+msgid "One bottle"
+msgid_plural "{0} bottles"
+msgstr[0] "eine Flasche"
+msgstr[1] "{0} Flaschen"
+
+#: src//puppetlabs/i18n/main.clj:27
 msgid "It took {0} programmers {1} months to implement this"
 msgstr "{0} Programmierer brauchten {1} Monat(e) um das zu implementieren"
 
-#: src/puppetlabs/i18n/main.clj:27
+#. first item is number of programmers, second is number of months
+#: src//puppetlabs/i18n/main.clj:35
+msgid "It took {0} {1} to implement this"
+msgstr "{0} {1} um das zu implementieren"
+
+#: src//puppetlabs/i18n/main.clj:36
+msgid "one programmer"
+msgid_plural "{0} programmers"
+msgstr[0] "ein Programmierer"
+msgstr[1] "{0} Programmierer"
+
+#: src//puppetlabs/i18n/main.clj:37
+msgid "one month"
+msgid_plural "{0} months"
+msgstr[0] "ein Monat"
+msgstr[1] "{0} Monate"
+
+#: src//puppetlabs/i18n/main.clj:41
```

Comment:
This example is where interpolating text into other text gets tricky. The correct sentences in German would be
```
Ein Programmierer brauchte einen Monat um das zu implementieren
Ein Programmierer brauchte 5 Monate um das zu implementieren
3 Programmierer brauchten einen Monat um das zu implementieren
3 Programmierer brauchten 5 Monate um das zu implementieren
```
While you can still structure this as `"{0} brauchte(n) {1} um das zu implementieren"`, pluralizing the sentence based on the number of programmers, the correct translation for `one month` and `{0} months` depends on the sentence this gets interpolated into (yay for cases); for example, if the sentence was `"We are going on vacation in {0}"` the translation for `"5 months"` would now be `"5 Monaten"` rather than `"5 Monate"` in the previous example (Dativ vs Akkusativ for the grammar geeks ;) ).

I am pretty sure that these sort of complications will be exceedingly rare in our code, and think the example should just be simplified to just have one thing in it that triggers pluralization.